src/envo/envo-edit.owl

```diff
@@ -3563,6 +3563,11 @@ Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_04000015>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_04000016>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_04000017>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_04000018>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_06105101>))
```

Comment:
Hi @iwilkie and @pbuttigieg , I think that the "plastic" term we added in #1080 is showing up again here. The #1080 PR has since been merged. I'm not sure if this could cause a conflict when merging this as well.

Reply:
Hmm - no merge conflicts were reported - did you update this branch with the commits merged into master from 1080?